fixture-generator/Dockerfile

```
@@ -16,43 +16,49 @@ RUN apt-get update && apt-get install -y \
-RUN git clone --depth 1 --branch ${KIBANA_VERSION} https://github.com/elastic/kibana.git . || \
-    git clone --depth 1 https://github.com/elastic/kibana.git .
-
 # Bootstrap Kibana (this takes 15-30 minutes)
 # This makes @kbn/lens-embeddable-utils and other packages available
 RUN yarn kbn bootstrap --allow-root
 
-# Create tool workspace
-WORKDIR /tool
-RUN mkdir -p /tool/output /tool/examples
+# Install tsx for running TypeScript directly (Kibana packages are not pre-compiled)
+RUN npm install -g tsx
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)